docs/DeveloperGuide.md

```diff
@@ -319,12 +317,93 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given index is invalid.
+**Use case: UC07 Filter by Tag**
 
-    * 3a1. AddressBook shows an error message.
+**MSS**
+
+1. User filters for contacts with a specified tag.
+2. WedLinker only shows a list of contact with the specified tag.
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+**Use case: UC08 Create Tags**
+
+**MSS**
+
+1. User creates a tag.
+2. WedLinker displays the successful creation of tag.
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. The tag already exists.
+* 2b. WedLinker does not create a new tag.
```

Comment:
Since these 2a, 2b, 2c are all part of the same extension, I think it should be 2a1, 2a2, 2a3?

Reply:
Thanks for catching that.